### ISSUE TITLE
Updating README

### ISSUE THREAD

**Issue by allthatilk** [AAMani5/digitalaffects#28]
README needs to be updated to include Laplacian smoothing and tokenizer info. Plus need to elaborate on the fact data used to choose Naive Bayes classifier was based on movie reviews, rather than tweets which are harder to classify. Eventually, want to include info for Docker too.

**Comment by allthatilk** [comment]
Info about chosen Naive Bayes version would be great too.

**Comment by AAMani5** [comment]
I agree that we need to be clear on what dataset was used to choose the classifier. Should we include the NLTK version as that readily available when compared to classifier version. 
will wait for @evebalog to give her input

**Comment by allthatilk** [comment]
Also might be good to mention somewhere that it's not optimised for mobile browsing.
